src/libraries/System.Runtime.Serialization.BinaryFormat/src/System/Runtime/Serialization/BinaryFormat/ObjectNullMultipleRecord.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Runtime.Serialization.BinaryFormat.Utils;
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+/// Represents multiple <see langword="null" />.
+/// </summary>
+/// <remarks>
+/// ObjectNullMultiple records are described in <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/f4abb5dd-aab7-4e0a-9d77-1d6c99f5779e">[MS-NRBF] 2.5.5</see>.
+/// </remarks>
+internal sealed class ObjectNullMultipleRecord : NullsRecord
+{
+    private ObjectNullMultipleRecord(int count) => NullCount = count;
+
+    public override RecordType RecordType => RecordType.ObjectNullMultiple;
+
+    internal override int NullCount { get; }
+
+    internal static ObjectNullMultipleRecord Decode(BinaryReader reader)
+    {
+        int count = reader.ReadInt32();
+        if (count <= 0)
+        {
+            ThrowHelper.ThrowInvalidValue(count);
+        }
+
+        return new ObjectNullMultipleRecord(count);
+    }
```

Comment:
While ObjectNullMultiple256 permits 0, ObjectNullMultiple does say "MUST be a positive integer"

Personally, I would have sprinkled comments all throughout these Decode methods, like

```suggestion
    internal static ObjectNullMultipleRecord Decode(BinaryReader reader)
    {
        // 2.5.5 ObjectNullMultiple
        
        // NullCount (4 bytes): An INT32 value ... The value MUST be a positive integer.
        int count = reader.ReadInt32();
        if (count <= 0)
        {
            ThrowHelper.ThrowInvalidValue(count);
        }

        return new ObjectNullMultipleRecord(count);
    }
```

As seen in https://github.com/dotnet/runtime/blob/db0eb5da5024d93dd976708c72fb93f7971e6f10/src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnDecoder.UtcTime.cs#L68-L83 or https://github.com/dotnet/runtime/blob/db0eb5da5024d93dd976708c72fb93f7971e6f10/src/libraries/Common/src/System/Security/Cryptography/Pkcs12Kdf.cs#L85-L119

That would explain where things like rejecting 0 come from.  They shouldn't be added in this PR (because it's already a bit excessive for the GH UI); but should you feel inspired I think it would be a happy followup.